firedrake/slate/slate.py

```diff
@@ -393,6 +400,15 @@ def __new__(cls, function):
             raise TypeError("Expecting a Coefficient or AssembledVector (not a %r)" %
                             type(function))
 
+    @cached_property
+    def terminal(self):
+        """Returns False unless the node is a terminal node."""
+        return True
```

Comment:
This kind of class-level property can just be written as `terminal = True`

Reply:
AH, yes. Can that be cached, too?

Reply:
there's no need, there's just one class-level attribute (rather than an instance attribute). So:

```
class Foo:
   terminal = True

x = Foo()
x.terminal => True
```

Reply:
Ah right